### ISSUE TITLE
Make angeltypes hidable from angeltype overview

### ISSUE THREAD

**Issue by promasu** [engelsystem/engelsystem#1379]
We have currently different attributes to hide an angeltype from the dashboard or shift overview for example.
Currently there is no way to hide a angeltype from the angeltype overview, like you would do for finance people, project lead etc.

That would be a nice feature to have to avoid confusion.

cc @jcgruenhage

**Comment by xuwhite** [comment]
This would contradict the principle of transparency towards all users to make all work visible. "hide on shift view" and "hide on registration" exist to simplify views for standard users.

**Comment by MyIgel** [comment]
I agree with xu on that point, we should not hide any more shifts from users. Eventually we could use the "hide on angeltypes" flag to not show the join button on the list and/or display the line in gray but adding another feature that actively hides it would imply much more sideeffects (like also having to hide shifts etc).

**Comment by jcgruenhage** [comment]
What benefit is there in making the shifts, location and who is working that shift public, if the team is fully staffed and the shifts are all filled?

I'd strongly prefer if there was a proper visibility system in place, where you could show shifts, rooms, angel types and basically all other types of resources to only those people that have a specific angel type set.

I know that there is a conflict of interest there, but denying the feature based on transparency requirements is only going to result in more tools being used for the same job, reducing the transparency further.

**Comment by MyIgel** [comment]
Thats a system that would be too complex for the current implementation so will not happen in the near future
